tests/test_unix_socket.py

```diff
@@ -162,11 +166,11 @@ async def client(app, loop):
 app_multi = Sanic(name="test")
 
 
-def handler(request):
+def handler(request: Request):
     return text(f"{request.conn_info.server}")
 
 
-async def client(app, loop):
+async def client(app: Sanic, loop: AbstractEventLoop):
```

Comment:
There really is no reason to start annotating all of these. Especially those embedded handlers.